src/main/java/frc/robot/subsystems/IntakeSubsystem.java

```diff
@@ -66,6 +67,10 @@ public void stop() {
     spinWheels(0);
   }
 
+  public Command autonIntake() {
+    return new StartEndCommand(this::pull, this::stop, this).withTimeout(1);
```

Comment:
Do we really rely on our high auton sequence to complete in 1 second? Is there a better way to sequence it?

Reply:
Doesn't really rely on anything, just makes sure the cone is secure
Ill move it into the scoring sequence, so the arm continues to intake until it scores